### ISSUE TITLE
Beta release

### ISSUE THREAD

**Issue by paul-buerkner** [stan-dev/posterior#74]
As far as I see, all issues currently assigned to the beta release milestone have been closed, so I wanted to open the discussion of when do to the beta release of posterior, and what may need to be done beforehand. Personally, I am fine with doing the beta release now, and I would like to hear your opinions, @jgabry, @mjskay, @MansMeg, @rok-cesnovar, and @avehtari?

**Comment by jgabry** [comment]
I'm ok with that. Ideally I think we'd have one or several basic vignettes first because I think it's a bit hard to get a feel for the package from the function doc itself, but it's not a huge deal if we go ahead with a beta release and add the vignette(s) before the CRAN release.

**Comment by paul-buerkner** [comment]
I have extended the README a bit to introduce a lot of the newer methods as well, so I don't think we need vignettes for the beta-release. Of course, we should have at least one overview vignette vor the CRAN release.

**Comment by rok-cesnovar** [comment]
Let me know when to update the package on the stan r-packages. Or feel free to update yourself if you want to try it. If not, I am happy to.

**Comment by jgabry** [comment]
@paul-buerkner Just noticed something to fix before beta release: an error in the examples for `weight_draws` (see examples at http://mc-stan.org/posterior/reference/weight_draws.html).

**Comment by jgabry** [comment]
I'll open a separate issue

**Comment by jgabry** [comment]
Should we start a `news.md` file with the beta release as the first entry? It probably makes sense to start tracking changes after the beta release

**Comment by paul-buerkner** [comment]
Sounds good!

Jonah Gabry <notifications@github.com> schrieb am Do., 11. Juni 2020, 20:45:

**Comment by jgabry** [comment]
Ok I can go ahead and create the news file

**Comment by jgabry** [comment]
@paul-buerkner I just pushed the news file and also bumped the version number in description to from 0.0.4 to 0.1.0, which seems reasonable for the beta release. If you'd rather go with a different version number feel free to change it. 

Once you've tagged the release I'll go ahead and regenerate the pkgdown website with the new readme, news file, and version number.

**Comment by paul-buerkner** [comment]
I have tagged the beta-release (https://github.com/stan-dev/posterior/releases/tag/v0.1.0-beta).

**Comment by jgabry** [comment]
Awesome. I'll regenerate the website. Do you want to announce the release on discourse?

**Comment by jgabry** [comment]
website for 0.1.0 is up

**Comment by avehtari** [comment]
This went too fast. @paul-buerkner tagged me in the first message for an opinion, but I didn't have time to comment.

**Comment by paul-buerkner** [comment]
Sorry, Aki. I will wait until I announce the beta release until you had time to give feedback.

**Comment by jgabry** [comment]
> This went too fast. @paul-buerkner tagged me in the first message for an opinion, but I didn't have time to comment.

Good point, sorry! 

> Sorry, Aki. I will wait until I announce the beta release on Stan discourse until you had time to give feedback.

Yeah we can definitely make more changes if necessary before announcing a release.

**Comment by avehtari** [comment]
Ok, I had now time to check how it currently looks.

I suggest adding citation for the package in the readme and web page. Not everyone know about citation() and some people who are not even using R might just check our reference implementations for diagnostics.

Now the only reference is for the eight schools data. It would be good to cite the new Rhat ESS paper in readme and the web page in the section "Summarising draws". Something like
"Basically, we get a data frame with one row per variable and one column per summary statistic or convergence diagnostic. The summaries rhat, ess_bulk and ess_tail are described in Vehtari et al. (2020). ..."

> phi resembles (mu + tau)^2 (up to rounding two 2 digits

resembles -> is equal to 

> Provide lightweight implementations of state of the art MCMC diagnostics.

As the support for weighted draws was added, I assume we'll add support for IS diagnostics, and thus this could be

"Provide lightweight implementations of state of the art posterior inference diagnostics."

**Comment by paul-buerkner** [comment]
Thank you, Aki! I will implement these suggestions.

**Comment by paul-buerkner** [comment]
I have updated the citation info and made the other suggested changes to the README (which will be propagated to the website). @avehtari, can you double check the new info in the README?

**Comment by avehtari** [comment]
Looks good :+1:

**Comment by paul-buerkner** [comment]
Thank you! @jgabry can you update the website? Once that is done, I will make a post on discourse.

**Comment by jgabry** [comment]
Thanks Aki and Paul. Just updated the website so I think we're all set for you to post on discourse.

**Comment by paul-buerkner** [comment]
Good. Will prepare the post later on today.

**Comment by paul-buerkner** [comment]
@rok-cesnovar would you mind updating posterior on the stan R packages repo?

**Comment by rok-cesnovar** [comment]
I did already when Jonah said the website was ready.

**Comment by paul-buerkner** [comment]
Perfect thank you!

Rok Češnovar <notifications@github.com> schrieb am Fr., 12. Juni 2020,
18:56:

**Comment by paul-buerkner** [comment]
The release post can now be found at https://discourse.mc-stan.org/t/beta-release-of-our-posterior-r-package/15949
